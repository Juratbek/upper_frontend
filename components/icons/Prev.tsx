import { IIconProps } from './types';

export const PrevIcon = ({ width = 24, height = 24, color = 'black' }: IIconProps) => (
  <svg role='icon' width={width} height={height} viewBox='0 0 24 24' fill='none'>
    <path d='M15 6L9 12L15 18' stroke={color} strokeWidth='1.6' />
  </svg>
);
