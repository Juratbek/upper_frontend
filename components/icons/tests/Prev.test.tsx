import { render, screen } from '@testing-library/react';

import { PrevIcon } from '../Prev';

describe('Prev icon', () => {
  it('renders the icon', () => {
    render(<PrevIcon />);
    const icon = screen.getByRole('icon');
    expect(icon).toMatchSnapshot();
  });
});
