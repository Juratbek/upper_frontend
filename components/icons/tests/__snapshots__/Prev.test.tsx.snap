// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Prev icon > renders the icon 1`] = `
<svg
  fill="none"
  height="24"
  role="icon"
  viewBox="0 0 24 24"
  width="24"
>
  <path
    d="M15 6L9 12L15 18"
    stroke="black"
    stroke-width="1.6"
  />
</svg>
`;
